lib/bloc/fee_options/fee_options_bloc.dart

```diff
@@ -17,56 +17,93 @@ class FeeOptionsBloc extends Cubit<FeeOptionsState> {
   FeeOptionsBloc(this._breezSDK) : super(FeeOptionsState.initial());
 
   /// Fetches the current recommended fees
-  Future<List<FeeOption>> fetchFeeOptions(String toAddress) async {
+  Future<List<FeeOption>> fetchFeeOptions({required String toAddress, String? swapAddress}) async {
```

Comment:
I personally think it is better to group method by feature so `fetchRefundFeeOptions` is better to be inside the `refund_bloc` IMO.

Reply:
Agree, do we also wish to move fetchFeeOptions to sweep_bloc or keep it as is since both reverse_swap and sweep use it ?